### DISCUSSION TITLE
Como conectar uma planilha do sheets à página web?

### DISCUSSION THREAD

**Discussion by William-Albino** [community/community#141971] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Pessoal, boa noite, eu estou aprendendo a programar de maneira independente e estou tendo muita dificuldade de conectar uma planilha do Google sheets no meu código HTML, já tentei a API do sheetDB, ja tentei a API do Airtable e até com o Google scripts. Meu programa tem uma tela de login onde o usuário coloca a entrada (login e senha), compara com o email e senha da base de dados e abre um segundo HTML com as informações de uma outra tabela da memsa planilha (informações do usuário que logou), porém, nunca retorna, parece não carregar a planilha, nenhuma informação, só carrega a mensagem de boas vindas que eu deixei e mais nada. Tentando ir direto ao ponto hehe vocês sabem alguma API gratuita ou alguma maneira de eu subir planilhas, sejam elas do Google, Microsoft, Airtable, de maneira mais fácil?

**Comment by VishalKotekar** [comment]
To connect a Google Sheets spreadsheet to a web page, you can use the Google Sheets API or Google Apps Script. Here's a simple solution using Google Apps Script:

1. Go to your Google Sheets, click on Extensions > Apps Script.
2. Write a script to publish your data as a web service.
3. Use the URL generated by the script to fetch your spreadsheet data with JavaScript and display it on your web page.

This approach ensures smooth integration. Let me know if you need more guidance!
